Resources/views/frontend/index/index.tpl

```diff
@@ -43,7 +43,7 @@
                 source: autocomplete.sources.hits(index, {hitsPerPage: 10}),
                 displayKey: 'name',
                 templates: {
-                    {/literal}{if $showAlgoliaLogo}footer: '<div class="branding"><span><a href="https://github.com/synonymous1984/FroshAlgolia" target="_blank">FroshAlgolia Plugin by FriendsOfShopware</a></span><span>Powered by <img src="https://www.algolia.com/assets/algolia128x40.png" /></span> </div>',{/if}{literal}
+                    {/literal}{if $showAlgoliaLogo}footer: '<div class="branding"><span><a href="https://github.com/shyim/FroshAlgolia" target="_blank">FroshAlgolia Plugin by FriendsOfShopware</a></span><span>Powered by <img src="https://www.algolia.com/assets/algolia128x40.png" /></span> </div>',{/if}{literal}
```

Comment:
I'd suggest adding this image to the plugin to prevent a broken image, if the source doesn't exist anymore.

Reply:
sounds good :-)
see https://github.com/shyim/FroshAlgolia/pull/7/commits/72c0b3e58c52edd0978a4e3e5d2a6c4ba96b5152